lib/signs/actions/signs.js

```diff
@@ -50,36 +51,22 @@ export type SignActions = ActionType<typeof receivedGtfsEntities> |
 let nextSignId = 0
 let nextStopEntityId = 100
 
-export function createSign (
-  entity?: Stop | Route,
-  agency?: any
-) {
+export function createSign (agency?: any, ...entities: Array<GtfsStop | GtfsRoute>) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
-    nextSignId--
-    const entities = []
-
-    if (entity) {
-      nextStopEntityId++
-      // $FlowFixMe yes, stop_id may be undefined, that's why were checking it, flow!
-      const type = typeof entity.stop_id !== 'undefined' ? 'STOP' : 'ROUTE'
-      const newEntity = {}
-      newEntity.id = nextStopEntityId
-      newEntity.type = type
-
-      if (agency !== null) {
-        newEntity.agency = agency
-      }
-      const typeKey = type.toLowerCase()
-      newEntity[typeKey] = entity
-      entities.push(newEntity)
+    const affectedEntities = []
+    if (entities && entities.length > 0) {
+      entities.forEach(entity => {
+        nextStopEntityId++
+        affectedEntities.push(constructNewEntity(entity, agency, nextStopEntityId))
+      })
     }
-
+    nextSignId--
```

Comment:
Can you write some comments on how this works? Again... any way we can just use UUIDs here?